student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(name, classroom = 'unknown')
+    super(name)
```

Comment:
- [ ] Constructor extends parent's constructor by adding @classroom and a parameter for it and the duper needs to access the three parameters i.e `super(age, name, parent_permission)`. You can learn more [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)

N.B: same for the teacher's class too.